src/Compilers/Core/CodeAnalysisTest/Text/StringTextTest.cs

```diff
@@ -226,7 +225,7 @@ public void FromStream_CheckSum_BOM()
         {
             var bytes = new byte[] { 0xef, 0xbb, 0xbf, 0x61, 0x62, 0x63 };
 
-            var source = SourceText.From(new MemoryStream(bytes), Encoding.ASCII);
+            var source = SourceText.From(new MemoryStream(bytes), Encoding.UTF8);
```

Comment:
I think it was missing in Portable and my first pass at this was to get the stuff to Portable, not CoreCLR. I'll see what breaks when I try and reverse some of these changes.